### PR TITLE
Listing model kpugh

### PR DESCRIPTION
**Pull request by kpugh97**
adding fields to model

### CODE REVIEW

File: findmyparadise/src/main/java/com/tp/wrc/findmyparadise/persistence/ListingDAO.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.tp.wrc.findmyparadise.persistence;
+
+import com.tp.wrc.findmyparadise.models.Listing;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface ListingDAO extends JpaRepository<Listing, Integer> {
```
Comment (zmiles17) [inline findmyparadise/src/main/java/com/tp/wrc/findmyparadise/persistence/ListingDAO.java:6]: Isn't this the same as the ListingRepository?

**Review by zmiles17 — APPROVED** [review]
Looks good. Just a little confused as to why the ListingDAO was added.

File: findmyparadise_frontend/package-lock.json [hunk 2]
```diff
@@ -16436,6 +16436,16 @@
           "integrity": "sha512-nQyp0o1/mNdbTO1PO6kHkwSrmgZ0MT/jCCpNiwbUjGoRN4dlBhqJtoQuCnEOKzgTVwg0ZWiCoQy6SxMebQVh8A==",
           "dev": true
         },
+        "ansi-styles": {
```
Comment (wcollins37) [inline findmyparadise_frontend/package-lock.json:16439]: Pretty sure sure this file is supposed to be gitignored

**Review by wcollins37 — CHANGES_REQUESTED** [review]
Pretty good, just need to ignore package-lock.json and get rid of ListingDAO

**Pull request closed without merging**